modules/openapi-generator/src/main/resources/kotlin-spring/libraries/spring-boot/defaultBasePath.mustache

```diff
@@ -0,0 +1 @@
+{{contextPath}}
```

Comment:
Is `defaultBasePath.mustache` here for compatibility or extension reasons? If not, can `{{contextPath}}` just be inserted where `{{>defaultbasePath}}` exists, to simplify template maintenance and customization?

I don't see `{{>defaultbasePath}}` anywhere, and I haven't looked too in-depth into the template inclusions to understand if it's potentially in the missing file.